modules/web/js/ballerina/visitors/statement-visitor.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
copyright year should be 2016 or 2016-2017? I think we shouldn't change this (not sure).